src/Scene.tsx

```diff
@@ -328,35 +327,20 @@ export default function Scene(){
                  }
```

Comment:
About onUndoBtn function
1. In the body of condition (is object exist) should be only remove block function (Controls.removeCube(object));
2. Deleting of the last element from history should be below (like outside of condition's body about action).
There are right code for this function on the attched image
![image](https://user-images.githubusercontent.com/89405012/230679690-bd8f0f70-aef3-427a-a394-cb54b0868be7.png)
.

Reply:
the same comment applies here